app/tests/test_sessions.py

```diff
@@ -0,0 +1,65 @@
+import json
+from .base import SessionsBaseTestCase
+
+
+class SessionsTestCase(SessionsBaseTestCase):
+    def setUp(self):
+        super().setUp()
+        self.session_id = self.session["_id"]
+
+    def test_gets_session_answer_with_valid_id(self):
+        response = self.client.get(f"/sessions/{self.session_id}")
+        assert response.status_code == 200
+        session = response.json()
+        for key in ["quiz_id", "user_id"]:
+            assert session[key] == self.session[key]
+
+    def test_get_question_returns_error_if_id_invalid(self):
```

Comment:
`question`?

Reply:
I actually corrected everything, maybe it all got undone. Sorry,